source/features/deep-reblame.tsx

```diff
@@ -37,7 +42,11 @@ const getPullRequestBlameCommit = mem(async (commit: string, prNumber: number):
 	const associatedPR = repository.object.associatedPullRequests.nodes[0];
 
 	if (!associatedPR) {
```

Comment:
```suggestion
	if (associatedPR?.number !== prNumber || associatedPR.mergeCommit.oid !== commit) {
```

This way you can avoid the duplicate identical error

Reply:
Done